docs/modeling-your-data/modeling-your-data-with-dbt/migration-guides/utils/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Utils"
+sidebar_position: 100
+---
+
+### Upgrading to 0.14.0
+
+- Version 1.4.0 of `dbt-core` now required
```

Comment:
Shouldn't we warn users that if they use the BDP Console they won't be able to run their models for now (as on v.1.3 is available there)?

Reply:
I've got an open JIRA ticket with platform to add 1.4.4, should hopefully be live by the time we release